service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/SendNoticeOfHearingHandlerTest.java

```diff
@@ -90,7 +91,7 @@ void shouldSendNotificationToLAWhenNewHearingIsAdded() {
             LocalAuthorityInboxRecipientsRequest.builder().caseData(caseData).build()))
             .willReturn(Set.of(LOCAL_AUTHORITY_EMAIL_ADDRESS));
```

Comment:
@tomaszpowroznik are there similar tests for verifying that letters are sent to LA Legal Representative, Barrister, CAFCASS Solicitor and CAFCASS Guardian as well as emails?

Reply:
thanks @tomaszpowroznik for the call explaining that these tests here are covering the code sufficiently. 👍